spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/metadata.scala

```diff
@@ -81,7 +81,9 @@ object metadata {
     AtlasTypeUtil.createUniqueRequiredAttrDef(
       "qualifiedName", new AtlasStringType),
     AtlasTypeUtil.createOptionalAttrDef("tableType", new AtlasStringType),
+    AtlasTypeUtil.createOptionalAttrDef("schemaDesc", new AtlasStringType),
     AtlasTypeUtil.createOptionalAttrDef("provider", new AtlasStringType),
+    AtlasTypeUtil.createOptionalAttrDef("partitionProvider", new AtlasStringType),
```

Comment:
As mentioned by @arunmahadevan these are not needed as they would be covered in the models json

Reply:
@aadarshjajodia
Let us just make the change here so that we are not blocked by Atlas model review. We can do it concurrently, and eventually remove the model definitions in SAC when we are 100% sure Atlas provides these by default.